src/main/java/FrameworksDrivers/ScheduleGraphics.java

```diff
@@ -205,23 +246,14 @@ public void actionPerformed(ActionEvent e) {
 
                 Event event = new Event(eventName, eventDate, eventStartInt, eventEndInt);
                 try {
-                    manager.addEvent(event, user);
+                    program.addEvent(event, user);
```

Comment:
i think the way "addEvent" works has been changed to have different parameters since this was written